scripts/oom_handler.py

```
@@ -376,6 +376,10 @@ def main(args):
     oom_dt    = datetime.now(ZoneInfo("Asia/Jerusalem"))
     timestamp = oom_dt.strftime("%Y-%m-%d %H:%M:%S")
 
+    if args.pod == "node-f-0":
+        print(f"ℹ️  Skipping OOM handler for pod 'node-f-0'.")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)